### ISSUE TITLE
Bug: Parameters field in the PathItem is ignored

### ISSUE THREAD

**Issue by kamijin-fanta** [Himenon/openapi-typescript-code-generator#47]
## Steps To Reproduce

https://himenon.github.io/openapi-typescript-code-generator-playground/v0.8.0/index.html?code=PYBwpgdghiCWBcACAzAOgAyuQKFhAZsPNoogG5gBOAzrMBEgIwYYmIAus7ANmEgCIBRALIB5NgBMw1AMaVYITvSRSAtsETVwMtt1gzI1Pm1LRVfRMICSAFWzYjlCjWKkAtIgCulbkgBEABbs7CDwAPRhUmSoMLCo7NLsqBLAqlB4YX4miFKy8op0DIj8YBTcoOYQ7IiCEGSwlPSV7Gwe3r6IgcEg1OFhsfGJyanpEJnZuXIKSkUACo0SnjIzNXUNTZAt2DKpIPSbvWyyAWBph6SkAIKe7AHAlK4XpOwAnuBIwABGAFZgy9mkShgACOngaYAkjye7kQsAkAIuHjMYARIEa4EonGkUOhcJx0I4bws1HY8ggAHMEaYoOZ8dDXu9NKS8JSCU9JvkZkgoDc7pREMjsgAhYDAADWdIZFi%2Bv3%2BBI500KSBF4sswCk3ARQNB4MhVMQHk4PBRbINiB5t3u%2Bo8VgAykKAHLWxAgTyfPTUAKXFqmtogCRQBISb2o9FULHnNlG3h0p5SpAksmstkWvmxi4AEiB%2BH8AGIwjtVHsIAcwsdTlBqGFrpbKFlTXbHennkSE8yKfrXe7YJ7vc3CYzEyz9aRCJQ0uwVIGTWzPP7p8HJyOB8T28nTWOJ1OEmwgVp6EZI6QVRKEQqCspEABxMDVE8InZVTbpmAgPQyQOFMLfajKZfltJ%2B1ILMwBzTp80LYtSwAyswhPLJ7BAQMAkjMJyVvMJPlFMUqwAbzhABfHEkMoGlbyoI8zWRJA4QRPAkCQ24tRBMEgUhDhKE8Gd2WkKYLyKE9ECsfgERg9N4yZJN9U3QMkE8MF4SedClwJUAqE-egrHY5STyFF4tOY-cIEPdMACZ0HQfsQLAvwIN2fYqirPdi0PODsOoLIgA

## The current behavior

The parameters field contained in the Path Item Object is ignored.

## The expected behavior

Processed in the same as the parameters included in the Operation Object.

**Comment by Himenon** [comment]
* [Playground](https://himenon.github.io/openapi-typescript-code-generator-playground/v0.8.0/index.html?code=PYBwpgdghiCWBcACAzAOgAyuQKFhAZsPNoogG5gBOAzrMBEgIwYYmIAus7ANmEgCIBRALIB5NgBMw1AMaVYITvSRSAtsETVwMtt1gzI1Pm1LRVfRMICSAFWzYjlCjWKkAtIgCulbkgBEABbs7CDwAPRhUmSoMLCo7NLsqBLAqlB4YX4miFKy8op0DIj8YBTcoOYQ7IiCEGSwlPSV7Gwe3r6IgcEg1OFhsfGJyanpEJnZuXIKSkUACo0SnjIzNXUNTZAt2DKpIPSbvWyyAWBph6SkAIKe7AHAlK4XpOwAnuBIwABGAFZgy9mkShgACOngaYAkjye7kQsAkAIuHjMYARIEa4EonGkUOhcJx0I4bws1HY8ggAHMEaYoOZ8dDXu9NKS8JSCU9JvkZkgoDc7pREMjsgAhYDAADWdIZFi%2Bv3%2BBI500KSBF4sswCk3ARQNB4MhVMQHk4PBRbINiB5t3u%2Bo8VgAykKAHLWxAgTyfPTUAKXFqmtogCRQBISb2o9FULHnNlG3h0p5SpAksmstkWvmxi4AEiB%2BH8AGIwjtVHsIAcwsdTlBqGFrpbKFlTXbHennkSE8yKfrXe7YJ7vc3CYzEyz9aRCJQ0uwVIGTWzPP7p8HJyOB8T28nTWOJ1OEmwgVp6EZI6QVRKEQqCspEABxMDVE8InZVTbpmAgPQyQOFMLfajKZfltJ%2B1ILMwBzTp80LYtSwAyswhPLJ7BAQMAkjMJyVvMJPlFMUqwAbzhABfHF0KXAlQCoT96CsSFEBIk8hReajUSgSgaVvKgj2hJE2KQOFlzwJAkNuZdtTBIEaNJTwZ1Nc8uUQE9ECsfh-xkE5AOXFtBzXDTEE3QMkE8MF4QJPdi0PdMACZ0HQfsQLAvwIN2fYqirUyD2kODsOoLIgA)

## Before

```yml
paths:
  /get/books/{id}:
    parameters:
    - name: id
      in: path
      required: true
      description: Book ID
      schema:
        type: string
        format: uuid
    get:
      operationId: getBookById
      responses:
        200:
          $ref: "#/components/responses/Books"
```

## After

Since [`parameters`](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#parameterObject) is an [Operation Object](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#operationObject), it seems that it belongs to [Paths](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#paths-object) differently.

```yml
paths:
  /get/books/{id}:
    get:
      operationId: getBookById
      parameters: // <----- here
        - name: id
          in: path
          required: true
          description: Book ID
          schema:
            type: string
            format: uuid
      responses:
        200:
          $ref: "#/components/responses/Books"
```

**Comment by Himenon** [comment]
> A list of parameters that are applicable for this operation. If a parameter is already defined at the Path Item, the new definition will override it but can never remove it. The list MUST NOT include duplicated parameters. A unique parameter is defined by a combination of a name and location. The list can use the Reference Object to link to parameters that are defined at the OpenAPI Object's components/parameters.

**Comment by Himenon** [comment]
https://github.com/Himenon/openapi-typescript-code-generator/issues/47#issuecomment-827387401 is not wrong, but the implementation just doesn't seem to be faithful.

**Comment by Himenon** [comment]
[fixed: v0.9.4 higher version](https://openapi-typescript-playground.netlify.app/v0.9.4/index.html?code=PYBwpgdghiCWBcACAzAOgAyuQKFhAZsPNoogG5gBOAzrMBEgIwYYmIAus7ANmEgCIBRALIB5NgBMw1AMaVYITvSRSAtsETVwMtt1gzI1Pm1LRVfRMICSAFWzYjlCjWKkAtIgCulbkgBEABbs7CDwAPRhUmSoMLCo7NLsqBLAqlB4YX4miFKy8op0DIj8YBTcoOYQ7IiCEGSwlPSV7Gwe3r6IgcEg1OFhsfGJyanpEJnZuXIKSkUACo0SnjIzNXUNTZAt2DKpIPSbvWyyAWBph6SkAIKe7AHAlK4XpOwAnuBIwABGAFZgy9mkShgACOngaYAkjye7kQsAkAIuHjMYARIEa4EonGkUOhcJx0I4bws1HY8ggAHMEaYoOZ8dDXu9NKS8JSCU9JvkZkgoDc7pREMjsgAhYDAADWdIZFi%2Bv3%2BBI500KSBF4sswCk3ARQNB4MhVMQHk4PBRbINiB5t3u%2Bo8VgAykKAHLWxAgTyfPTUAKXFqmtogCRQBISb2o9FULHnNlG3h0p5SpAksmstkWvmxi4AEiB%2BH8AGIwjtVHsIAcwsdTlBqGFrpbKFlTXbHennkSE8yKfrXe7YJ7vc3CYzEyz9aRCJQ0uwVIGTWzPP7p8HJyOB8T28nTWOJ1OEmwgVp6EZI6QVRKEQqCspEABxMDVE8InZVTbpmAgPQyQOFMLfajKZfltJ%2B1ILMwBzTp80LYtSwAyswhPLJ7BAQMAkjMJyVvMJPlFMUqwAbzhABfHEkMoGlbyoI8zWRJA4QRPAkCQ24tRBMEgUhDhKE8Gd2WkKYLyKE9ECsfgERg9N4yZJN9U3QMkE8MF4SedClwJUAqE-egrHY5STyFF4tOY-cIEPdMACZ0HQfsQLAvwIN2fYqirPdi0PODsOoLIgA)


Thanks !
